lumicks/pylake/force_calibration/calibration_models.py

```diff
@@ -235,19 +346,29 @@ def calibration_results(self, fc, diffusion_constant_volts, f_diode, alpha):
         """
         reference_peak = self(self.driving_frequency, fc, diffusion_constant_volts, f_diode, alpha)
 
+        # Equation 14 from [6]
         power_exp = (self._response_power_density - reference_peak) * self._frequency_bin_width
+
+        # Equation 12 from [6]
         distance_response = np.sqrt(self._theoretical_driving_power(fc) / power_exp)  # m/V
 
+        # Equation 16 from [6]
         temperature_kelvin = scipy.constants.convert_temperature(self.temperature, "C", "K")
         k_temperature = scipy.constants.k * temperature_kelvin
-        gamma_0 = k_temperature / (distance_response ** 2 * diffusion_constant_volts)  # kg/s^2
-        kappa = 2.0 * np.pi * fc * gamma_0  # N/m
+        measured_drag_coeff = k_temperature / (distance_response ** 2 * diffusion_constant_volts)
```

Comment:
I think it's good to leave the unit comment in

Reply:
It felt a bit like duplicating the unit that's right below. Also the old one was wrong. It's `kg/s`.

Reply:
fair enough